packages/aws-cdk/lib/api/deployments/asset-publishing.ts

```diff
@@ -18,19 +18,7 @@ import { ToolkitError } from '../../toolkit/error';
 import type { SdkProvider } from '../aws-auth';
 import { Mode } from '../plugin';
 
-export interface PublishAssetsOptions {
-  /**
-   * Print progress at 'debug' level
-   */
-  readonly quiet?: boolean;
-
-  /**
-   * Whether to build assets before publishing.
-   *
-   * @default true To remain backward compatible.
-   */
-  readonly buildAssets?: boolean;
-
```

Comment:
Only a single internal call-site of `publishAssets` is left and it is not specifying these options.